app/queries/my_facilities/missed_visits_query.rb

```diff
@@ -22,17 +22,15 @@ def initialize(facilities: Facility.all, period: :quarter, last_n: 3)
   def patients
     @patients ||=
       Patient
-      .joins('INNER JOIN latest_blood_pressures_per_patients
-              ON patients.id = latest_blood_pressures_per_patients.patient_id')
-      .where(latest_blood_pressures_per_patients: { bp_facility_id: @facilities })
+      .where(registration_facility_id: @facilities)
       .where('recorded_at < ?', Time.current.beginning_of_day - REGISTRATION_BUFFER)
   end
 
   def patients_by_period
     @patients_by_period ||=
       @periods.map do |year, period|
         period_start = (@period == :quarter ? local_quarter_start(year, period) : local_month_start(year, period))
-        [[year, period], patients.where('patient_recorded_at < ?', period_start - REGISTRATION_BUFFER)]
+        [[year, period], patients.where('recorded_at < ?', period_start - REGISTRATION_BUFFER)]
```

Comment:
Has this changed because we no longer join with another table?

Reply:
Yup, the joined table had `patient_recorded_at` and `bp_recorded_at`. Need to use `recorded_at` now since we're just looking up Patient.